common/djangoapps/student/helpers.py

```diff
@@ -270,6 +262,31 @@ def get_next_url_for_login_page(request):
     return redirect_to
 
 
+def get_redirect_to(request):
+    """
+    Determine the redirect url and return if safe
+    :argument
+        request: request object
+
+    :returns: redirect url if safe else None
+    """
+    static_file_format = '\w+\.(jpeg|jpg|gif|png|ico)'
```

Comment:
@schenedx I see, thanks; I forgot to read the associated ticket. That does seem like something we'd like to prevent. Have you considered disallowing any redirect values that end with an extension? Legitimate redirects should just be relative paths with no extension (e.g., `/dashboard`).